### PR TITLE
Search filterOutput collection via "filter_id" or "id"

### PR DESCRIPTION
**Pull request by PauloPortugal**
### What

In CMD we search by `filter_id`, but on Cantabular we search by a new
field called `id`.

This addresses the approach of simply changing the MongoDB query.

Resolves: [5690](https://trello.com/c/ZDZgbMyz/5690-copy-getfilteroutput-function-from-dp-filter-api-and-amend-for-cantabular-filter-output-search)

### How to review

Review the changes and run them locally

### Who can review

Any ONS developer

### CODE REVIEW

File: models/models.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ type Filter struct {
 	LastUpdated     time.Time           `bson:"last_updated"               json:"-"`
 	ETag            string              `bson:"e_tag"                      json:"-"`
 
-	ID         string      `bson:"-"                    json:"id,omitempty"`
```
Comment (saminahbab) [inline models/models.go:50]: this is probably fine, as long as we let the stakeholders beyond team b know that we are going this far

File: mongo/filterstore.go [hunk 2]
```diff
@@ -377,9 +377,9 @@ func (s *FilterStore) CreateFilterOutput(ctx context.Context, filter *models.Fil
 
 // GetFilterOutput returns a filter output resource
 func (s *FilterStore) GetFilterOutput(ctx context.Context, filterID string) (*models.Filter, error) {
-	// NOTE: previously was filter_id.
-	// This should be _id/id. Confirm during PR.
-	query := bson.M{"filter_id": filterID}
+	// we have to match either the filter_id (in CMD) or the id (in Cantabular)
```
Comment (saminahbab) [inline mongo/filterstore.go:380]: I think a slightly longer comment explicitly saying why it led to having to support in this way would be helpful for future context

**Review by j-s-rawat — APPROVED** [review]
LGTM

File: models/models.go (hunk 1 above)
Comment (PauloPortugal) [reply models/models.go:50]: @franmoore05 @eldeal :point_up_2:

**Pull request merged**